docs/tutorials/tictactoe.rst

```diff
@@ -428,6 +428,7 @@ With the above preparation, we are close to the first learned agent. The followi
         agent_opponent: Optional[BasePolicy] = None,
     ) -> None:
         env = get_env()
+        env = DummyVectorEnv([lambda: env])
```

Comment:
In collector, it will automatically wrap the single instance env to vector env, see https://github.com/thu-ml/tianshou/blob/278c91a2228a46049a29c8fa662a467121680b10/tianshou/data/collector.py#L65-L67
So you should modify this line instead of documentation